x-pack/plugins/spaces/public/copy_saved_objects_to_space/components/space_result_details.scss

```diff
@@ -11,3 +11,15 @@
   // Constrains name to the flex item, and allows for truncation when necessary
   min-width: 0;
 }
+
+.spcCopyToSpaceResultDetails__selectControl {
+  margin-left: 30px;
```

Comment:
Seems like an arbitrary number, any reason not to use an `$eui` variable?

Reply:
Yep, that was arbitrary, and it was only meant to be a placeholder. Thanks for catching it, I'll change it to `$euiSizeL`.